argilla-server/src/argilla_server/database.py

```diff
@@ -38,6 +38,7 @@
         "2.0": "237f7c674d74",
         "2.4": "660d6c6b3360",
         "2.5": "6ed1b8bf8e08",
+        "2.x": "580a6553186f",
```

Comment:
We should replace the `2.5` tag right?

```suggestion
        "2.5": "580a6553186f",
```

Reply:
I think this may be included for a future release, not the 2.5. I think it will be the 2.6.0